package.json

```diff
@@ -45,6 +45,7 @@
     "@nrwl/tao": "13.0.1",
     "@nrwl/workspace": "13.0.1",
     "@octokit/core": "^3.2.1",
+    "@openapitools/openapi-generator-cli": "^2.5.1",
```

Comment:
Is this for having `npx` use the locally installed package during e2e tests?

Of course I agree with the general idea, but I'd rather have the tests use the exact same version we use when the plugin is installed by a user.

See: `packages/nx-plugin-openapi/src/utils/versions.ts` 

Since I don't like the thought of having to keep the version number aligned in two places, I'd rather not install this devDependency here and instead have the e2e test file install it using the version from `packages/nx-plugin-openapi/src/utils/versions.ts` before launching the executor.

Reply:
that makes sense, i'll change that